### PR TITLE
docs: Update FEATURES.md

### PR DESCRIPTION
**Pull request by emmambd**
**Summary:**

Add associated gtfs.org documentation link to the features markdown.

### CODE REVIEW

**Comment by emmambd** [comment]
Setting to draft until 6.0 is released. cc @Sergiodero for review. 

Going through this also made me realize: should `Frequencies` be called [`Frequency-Based Service`](https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service)? You said it could be the same in #1679 but it looks like the name change would be the 1:1 mapping

**Comment by Sergiodero** [comment]
> Going through this also made me realize: should Frequencies be called [Frequency-Based Service](https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service)? You said it could be the same in https://github.com/MobilityData/gtfs-validator/issues/1679 but it looks like the name change would be the 1:1 mapping

@emmambd I probably missed it! you're right, `Frequency-Based Service` should be the correct name.

File: docs/FEATURES.md [hunk 1]
```diff
@@ -1,32 +1,75 @@
...
+| Accessibility             | Text-to-speech            | One **tts_stop_name** value in [stops.txt](https://gtfs.org/schedule/reference/#stopstxt)  |   https://gtfs.org/getting_started/features/accessibility/#text-to-speech |
+| Accessibility             | Stops Wheelchair Accessibility   | One **wheelchair_boarding** value in [stops.txt](https://gtfs.org/documentation/schedule/reference/#stopstxt) |  https://gtfs.org/getting_started/features/accessibility/#stops-wheelchair-accessibility |
+| Accessibility             | Trips Wheelchair Accessibility   | One **wheelchair_accessible** value in [trips.txt](https://gtfs.org/documentation/schedule/reference/#tripstxt) |   https://gtfs.org/getting_started/features/accessibility/#trips-wheelchair-accessibility |
+| Base add-ons            | Route Colors              | One **color** value <br>OR<br>one text_color value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)    |    https://gtfs.org/getting_started/features/base_add-ons/#route-colors |
```
Comment (jcpitre) [inline docs/FEATURES.md:10]: Is it possible that `color` has been changed to `route_color` and `text_color` changed to `route_text_color` in routex.txt?

Comment (Sergiodero) [reply docs/FEATURES.md:10]: On the spec side I believe they have always been named as `route_color` and `route_text_color`

File: docs/FEATURES.md [hunk 2]
```diff
@@ -1,32 +1,75 @@
...
+| Fares                    | Fares Transfers           | One line of data in [fare_transfer_rules.txt](https://gtfs.org/schedule/reference/#fare_transfer_rulestxt)    |    https://gtfs.org/getting_started/features/fares/#fares-transfers |
+| Fares                    | Fares V1                  | One line of data in [fare_attributes.txt](https://gtfs.org/schedule/reference/#fare_attributestxt)   |   https://gtfs.org/getting_started/features/fares/#fares-v1 |
+| Pathways                 | Pathways Connections*         | One line of data in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt)     |  https://gtfs.org/getting_started/features/pathways/#pathway-connections |
+| Pathways                 | Pathways Details*         | One value of **max_slope**<br/>OR<br/>**min_width** <br/>OR<br/>**length** <br/>OR<br/>**stair_count** in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt) | https://gtfs.org/getting_started/features/pathways/#pathway-details |
```
Comment (Sergiodero) [inline docs/FEATURES.md:28]: Same as with Pathway Connections, the name used here should be Pathway Details (with the word "pathways" changing to singular). Also correct in the Feature Migration table below.

File: docs/FEATURES.md [hunk 3]
```diff
@@ -1,32 +1,75 @@
...
+| Fares                    | Route-Based Fares         | One **network_id** value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)<br/>OR<br/>one line of data in [networks..txt](https://gtfs.org/schedule/reference/#networkstxt)   |    https://gtfs.org/getting_started/features/fares/#route-based-fares   |
+| Fares                    | Time-Based Fares          | One line of data in [timeframes.txt](https://gtfs.org/schedule/reference/#timeframestxt)   | https://gtfs.org/getting_started/features/fares/#time-based-fares  |
+| Fares                    | Zone-Based Fares          | One line of data in [areas.txt](https://gtfs.org/schedule/reference/#areastxt)   |     https://gtfs.org/getting_started/features/fares/#zone-based-fares |
+| Fares                    | Fares Transfers           | One line of data in [fare_transfer_rules.txt](https://gtfs.org/schedule/reference/#fare_transfer_rulestxt)    |    https://gtfs.org/getting_started/features/fares/#fares-transfers |
```
Comment (Sergiodero) [inline docs/FEATURES.md:25]: @tzujenchanmbd and I had a discussion about this one and decided to change it to `Fare Transfers` (first word singular and second word plural). We'll update the documentation in GTFS.org shortly to reflect this. The name should also be updated in line 59 (in the Feature Migration List).

File: docs/FEATURES.md [hunk 4]
```diff
@@ -1,32 +1,75 @@
...
+|Route Colors|Route Colors|Route Colors|
+|Bike Allowed||Bike Allowed|Bike Allowed|
+|Translations|Translations|Translations|
+|Headsigns|Headsigns|Headsigns|
+|Fare Products|Fare Products|Fare Products|
+|Fare Media|Fare Media|Fare Media|
+|Route-Based Fares|Route-Based Fares||
+|Time-Based Fares|Time-Based Fares||
+|Zone-Based Fares|Zone-Based Fares|Zone-Based Fares|
+|Fares Transfers|Transfer Fares||
```
Comment (Sergiodero) [inline docs/FEATURES.md:59]: 6.0: `Fare Transfers`

File: docs/FEATURES.md [hunk 5]
```diff
@@ -1,32 +1,75 @@
...
+| Base add-ons                | Feed Information          | One line of data in [feed_info.txt](https://gtfs.org/schedule/reference/#feed_infotxt)  |  https://gtfs.org/getting_started/features/base_add-ons/#feed-information |
+| Base add-ons                | Attributions              | One line of data in [attributions.txt](https://gtfs.org/schedule/reference/#attributionstxt) |   https://gtfs.org/getting_started/features/base_add-ons/#attributions|
+| Base add-ons                  | Shapes                    | One line of data in [shapes.txt](https://gtfs.org/schedule/reference/#shapestxt)      |   https://gtfs.org/getting_started/features/base_add-ons/#shapes |   
+| Base add-ons               | Transfers                 | One line of data in [transfers.txt](https://gtfs.org/schedule/reference/#transferstxt)     | https://gtfs.org/getting_started/features/base_add-ons/#transfers  |
+| Base add-ons | Frequency-Based Service | One line of data in [frequencies.txt](https://gtfs.org/schedule/reference/#frequenciestxt)    | https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service  |
```
Comment (Sergiodero) [inline docs/FEATURES.md:19]: Name should be `Frequencies` (the correct name is already being used in the validator). I also realize we're being inconsistent in the documentation when referring to this feature, I'll update GTFS.org to use Frequencies across the board.

File: docs/FEATURES.md [hunk 6]
```diff
@@ -1,32 +1,75 @@
...
+| Pathways                 | Levels                    | One line of data in [levels.txt](https://gtfs.org/schedule/reference/#levelstxt)     |   https://gtfs.org/getting_started/features/pathways/#levels  |
+| Pathways                 | In-station traversal time | One **traversal_time** value in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt)   |   https://gtfs.org/getting_started/features/pathways/#in-station-traversal-time                                                                                                                                                                                                                                                                                                                                            |
+| Pathways                 | Pathway Signs       | One **signposted_as** value<br/>AND<br/>one **reversed_signposted_as** in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt) |       https://gtfs.org/getting_started/features/pathways/#pathway-signs     |
```
Comment (Sergiodero) [inline docs/FEATURES.md:31]: This rule should be: "One `signposted_as` value **OR** one `reversed_signposted_as` in `pathways.txt`

**Review by Sergiodero — COMMENTED** [review]
I left a few comments suggesting some adjustments, hopefully they are simple/clear enough, but LMK if you have any questions :)

File: docs/FEATURES.md (hunk 5 above)
Comment (emmambd) [reply docs/FEATURES.md:19]: We just changed this so we'll change it back!

**Review by Sergiodero — APPROVED** [review]
Changes LGTM, thanks for working on this and sorry for all the back and forth!

**Pull request merged**